<?xml version="1.0"?>
<launch>
  <arg name="state_vehicle_file_name" default="eurecar_state_vehicle.yaml" />
  <arg name="state_behavior_file_name" default="eurecar_state_behavior.yaml" />


  <node pkg="eurecar_decision_maker" type="eurecar_decision_maker_node" name="eurecar_decision_maker" output="screen">
    <param name="state_vehicle_file_name" value="$(find eurecar_decision_maker)/cfg/$(arg state_vehicle_file_name)" />
    <param name="state_behavior_file_name" value="$(find eurecar_decision_maker)/cfg/$(arg state_behavior_file_name)" />

  </node>
</launch>
